test/index.js

```diff
@@ -1163,6 +1163,41 @@ describe('either', function() {
 
   });
 
+  describe('maybeToEither', function() {
+
+    it('is a binary function', function() {
+      eq(typeof S.either, 'function');
+      eq(S.maybeToEither.length, 2);
+    });
+
+    it('type checks its arguments', function() {
+      assert.throws(function() { S.maybeToEither('left', 1); },
+                    errorEq(TypeError,
+                            '‘maybeToEither’ requires a value of type Maybe ' +
+                            'as its second argument; received 1'));
+
+      assert.throws(function() { S.maybeToEither(R.__, 1)('left'); },
+                    errorEq(TypeError,
+                            '‘maybeToEither’ requires a value of type Maybe ' +
+                            'as its second argument; received 1'));
```

Comment:
Let's drop this assertion. The `S.maybeToEither('left', 1)` test should be sufficient.